Landing-page.css

```diff
@@ -0,0 +1,842 @@
+    border-radius: 3rem;
+    background: url('img/svgs/Ellipse.svg');
+}
+
+.section-five .testimonies .testimony-2 .student-image {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 3rem;
+    background: url('img/svgs/Ellipse-1.svg');
+}
+
+.section-five .testimonies .testimony-3 .student-image {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 3rem;
+    background: url('img/svgs/Ellipse-2.svg');
+}
+
+
+.section-five .testimonies .testimony > div {
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    align-items: center;
+    margin-bottom: 1rem;
+}
+
+/***
+.section-five > div:nth-child(3) {
+    margin: 4rem;
+}
+
+.section-five > div:nth-child(3) h2,
+.section-five > div:nth-child(3) p,
+.section-five > div:nth-child(3) a
+ {
+    margin: 2rem;
+}
+**/
+
+.section-five > div:last-child {
+    width: 70vw;
+    padding-top: 4rem;
+    margin: auto;
+}
+
+.section-five > div:last-child p{
+    margin-bottom: 2rem;
+}
+
+
+
+
+
+
+
+
+/** 
+* FOOTER
+**/
+
+.section-six {
+    background: #051442;
+    padding: 2rem 0rem;
+}
+
+.section-six h2 {
+    padding-bottom: 2rem;
+}
+
+
+.section-six .footer-links-wrap {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 2rem;
+}
+
+.section-six li {
+    line-height: 2.0;
+}
+
+.section-six a:hover {
+    opacity: 0.7;
+}
+
+.section-six li,
+.section-six a {
+    color: #fff;
+    font-size: 0.9rem;
+
+}
+
+.horizontal-rule {
+    background-color: #000;
+    height: .15rem;
+}
+
+footer {
+    background: #041139;
+    padding: 2rem 0;
+}
+
+
+
+
+
+/** 
+* UTILITIES
+**/
+
+/* -----
+SVG Icons - svgicons.sparkk.fr
+----- */
+/*********
+.svg-icon {
+    width: 1em !important;
+    height: 1em !important;
+}
+  
+.svg-icon path,
+.svg-icon polygon,
+.svg-icon rect {
+    fill: #4691f6 !important;
+}
+  
+.svg-icon circle {
+    stroke: #4691f6 !important;
+    stroke-width: 1 !important;
+}
+
+********************/
+
+
+
+/** 
+* COLORS
+**/
+
+.bg-black {
+    background: #2F2E41;
+}
+
+.bg-blue {
+    background: #1F59BB !important;
+}
+/** 
+* BUTTONS
+**/
+
+.btn-md {
+    margin: 0 0.5rem;
+    padding: 1.0rem 2.0rem;
+    border-radius: 0.5rem;
+}
+
+.btn-md:hover {
+    opacity: 0.92;
+}
+
+.btn-lg {
+    padding: 1.5rem 3.0rem;
+    border-radius: 0.5rem;
+}
+
+.btn-lg:hover {
+    opacity: 0.92;
+}
+
+
+/** 
+* TEXTS (xs - extra small,
+         sm - small,
+         md - medium,
+         lg - large,
+         xl - extra large)
+**/
+
+/** 
+* TEXT - size
+**/
+
+
+.text-sm {
+    font-size: 0.9rem;
+}
+
+.text-md {
+    font-size: 1.2rem;
+}
+
+.text-lg {
+    font-size: 1.5rem
+}
+
+.text-xl {
+    font-size: 2.0rem;
+}
+
+/** 
+* TEXT - color
+**/
+
+.text-white {
+    color: #fff;
+}
+
+.text-black {
+    color: #000;
+}
+
+.text-center {
+    text-align: center;
+}
+
+/**
+** MEDIA QUERIES
+**/
+@media(max-width: 1075px) {
+    header .header-rt a.btn-md {
+        padding: .5rem !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575014)